spec/controllers/zines_controller_spec.rb

```diff
@@ -39,7 +37,16 @@
         it 'is decorated with ZineDecorator' do
           expect(assigns(:zine)).to be_decorated_with ZineDecorator
         end
+      end
+    end
+
+    context 'without any zines' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.